src/js/api.ts

```diff
@@ -683,6 +683,24 @@ export class PyodideAPI {
     }
     return API.makeSnapshot(serializer);
   }
+
+  /**
+   * Returns the `info` section of the pyodide lockfile used to load the current Pyodide instance.
+   * The format of the lockfile is defined in the `pyodide/pyodide-lock
+   * <https://github.com/pyodide/pyodide-lock`_ repository.
+   */
+  static getLockfileInfo() {
+    return API.lockfile_info;
+  }
+
+  /**
+   * Returns the `packages` section of the pyodide lockfile used to load the current Pyodide instance.
+   * The format of the lockfile is defined in the `pyodide/pyodide-lock
+   * <https://github.com/pyodide/pyodide-lock`_ repository.
+   */
+  static getLockfilePackages() {
+    return API.lockfile_packages;
+  }
```

Comment:
Perhaps there is a name that sounds more related to `loadPackages`, `loadedPackages`? These are the packages that can be loaded by `loadPackages`. But maybe `lockfilePackages` is fine.